toolchains/iOS.framework.cmake

```diff
@@ -3,7 +3,7 @@ include(${CMAKE_SOURCE_DIR}/toolchains/iOS.toolchain.cmake)
 message(STATUS "Build for iOS archs " ${CMAKE_OSX_ARCHITECTURES})
 
 set(FRAMEWORK_NAME TangramMap)
-set(FRAMEWORK_VERSION "1.0")
```

Comment:
OK, something like a `-beta`, `-alpha` or maybe `-dev` will probably be meaningful.